packages/story-editor/src/karma/fixture/fixture.js

```diff
@@ -230,9 +230,9 @@ export class Fixture {
     });
   }
 
-  restore() {
+  async restore() {
     window.location.hash = '#';
-    localStorage.clear();
+    await localStorage.clear();
```

Comment:
This is not an async function: https://developer.mozilla.org/en-US/docs/Web/API/Storage/clear